modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/provider/test/DefaultSegmentsEntryProviderTest.java

```diff
@@ -65,6 +70,39 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@Test
+	public void testGetOrganizationSegmentsWithDateModified() throws Exception {
+		_user1 = UserTestUtil.addUser(_group.getGroupId());
+		_user2 = UserTestUtil.addUser(_group.getGroupId());
+
+		_user2.setModifiedDate(_getDateBefore(_user1));
+
+		Organization organization = OrganizationTestUtil.addOrganization();
+
+		_user1 = UserTestUtil.addOrganizationUser(
+			organization, RoleConstants.ORGANIZATION_USER);
+
+		Criteria criteria = new Criteria();
+		String filterString = String.format(
+			"dateModified eq %s",
+			ISO8601Utils.format(_getDateModifiedTruncated(_user1)));
+
+		_userOrganizationSegmentsCriteriaContributor.contribute(
+			criteria, filterString, Criteria.Conjunction.AND);
+
+		SegmentsEntry segmentsEntry = SegmentsTestUtil.addSegmentsEntry(
+			_group.getGroupId(), CriteriaSerializer.serialize(criteria));
+
+		long[] segmentsEntryIds = _segmentsEntryProvider.getSegmentsEntryIds(
+			_group.getGroupId(), User.class.getName(), _user1.getUserId());
+
+		Assert.assertArrayEquals(
+			new long[] {segmentsEntry.getSegmentsEntryId()}, segmentsEntryIds);
+		Assert.assertEquals(
+			StringUtil.merge(segmentsEntryIds, StringPool.COMMA), 1,
+			segmentsEntryIds.length);
```

Comment:
@CaioERP, please group these line so that the variable declaration is as closer as possible from its usage